index.html

```diff
@@ -0,0 +1,177 @@
+<body>
+    <header>
+        <div class="logo">
+            <img class="mainLogo" src="Images/pngwing.com.png" alt="took from pngwing - Nerf">
+            <p class="logo__lyrics">Anime World</p>
+        </div>
+
+        <form class="search">
+            <div><input type="text" placeholder="Search" class="searchbar"></div>
+            <input type="button" value="" class="buttons"> 
+        </form>
+
+        <form class="menu">
+                <input type="button" value="" class="buttons option">
+                <input type="button" value="" class="buttons option">
+                <input type="button" value="" class="buttons option">           
+                <input type="button" value="" class="buttons option account"> 
+        </form>
+    </header>
+
+    <div class="page">
+
+        <article>
+            <div class="media">
+                <video src="video/RentaGirlfriend _Opening.mp4" controls></video>
+                <p class="title">Rent a Girlfriend opening</p>
+                <div class="media_info">
+                    <p>947.067 views - Feb 02 2022</p>
+                    <ul class="video_reaction">
+                        <input type="button" value="" class="buttons option">
+                        <input type="button" value="" class="buttons option">
+                        <input type="button" value="" class="buttons option">
+                        <input type="button" value="" class="buttons option">
+                    </ul>
+                </div>
+                <div class="credits">
+                    <div class="container">
+                        <img class="img_owner" src="Images/sasuke.png" alt="sasuke">
+                    </div>
+                    <ul class="credit_info">
+                        <li class="owner">Sasuke</li>
+                        <li class="owner subscriptions">5k subscriptors</li>
+                        <li class="owner data">Welcome to my anime world channel</li>
+                    </ul>
+                </div>
+            </div>
+        </article>
+
+        <aside>
+            <ul class="links">
+                <li><a href="#" class="links_buttons">All</a></li>
+                <li><a href="#" class="links_buttons">Anime</a></li>
+                <li><a href="#" class="links_buttons">Related</a></li>
+                <li><a href="#" class="links_buttons">Viewed</a></li>
+                <li><a href="#" class="links_buttons">More</a></li>
+            </ul>
+
+            <ul class="minivideos">
+                <li class="vidinfo"><img class="minimages" src="Images/images (1).jpeg" alt="img taken from google">
+                    <div class="mindetails">
+                        <a class="mintitle" href="#">Dragon Ball</a>
+                        <ul class="mininfo">
+                            <li>Uploaded by</li>
+                            <li>Visualization</li>
+                            <li>Date</li>
+                        </ul>
```

Comment:
Inside the list, you took some images. Please remove the spaces from the names of the images to accomplish the Linters standard procedures. Please apply these changes to the rest of the images.